### PR TITLE
ISSUE-345 made links downloadable without opening the original site

### PR DESCRIPTION
**Pull request by JyothiSwaroopReddy07**
Changed all links such that they directly download zip folders when clicked rather than getting redirected to apache websites

### CODE REVIEW

**Review by tsunghanjacktsai — APPROVED** [review]

**Comment by JyothiSwaroopReddy07** [comment]
I don't have write access to this repo? How do I merge my pull request

**Comment by mxsm** [comment]
> I don't have write access to this repo? How do I merge my pull request

Hi @JyothiSwaroopReddy07 ,Someone else with access will merge the RP,You just have to wait for PR to be merged.

**Pull request merged**
